src/Microsoft.DotNet.VersionTools/Dependencies/BuildOutput/DependencyBuildInfo.cs

```diff
@@ -7,9 +7,9 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Microsoft.DotNet.VersionTools.Dependencies
+namespace Microsoft.DotNet.VersionTools.Dependencies.BuildOutput
 {
-    public class DependencyBuildInfo
+    public class DependencyBuildInfo : IDependencyInfo
```

Comment:
I think this name should be flipped around.  This isn't a "BuildInfo".  This is a "DependencyInfo"

Reply:
Agreed, `BuildDependencyInfo` is fine by me.

If we wanted to a bit more descriptive, `BuildInfoDependencyInfo` would be the "full name", imo... but repeating `Info` like that seems bad. (Maybe `IDependencyInfo` (or `BuildInfo`) could use a better name.) I'm fine with `BuildDependencyInfo` so I'll stick with that unless someone has a problem with it.